recipes-parsec/parsec-service/parsec-service/config-softhsm.toml

```diff
@@ -0,0 +1,26 @@
+[core_settings]
+log_level = "trace"
+log_error_details = true
+
+[listener]
+listener_type = "DomainSocket"
+# The timeout needs to be smaller than the test client timeout (five seconds) as it is testing
+# that the service does not hang for very big values of body or authentication length.
+timeout = 3000 # in milliseconds
+
+[authenticator]
+auth_type = "UnixPeerCredentials"
+
+[[key_manager]]
```

Comment:
Why two brackets?

Reply:
Because that is the syntax that parsec requires. 

https://github.com/parallaxsecond/parsec/blob/main/config.toml#L85